saleor/product/management/commands/update_all_products_discounted_prices.py

```diff
@@ -2,22 +2,41 @@
 
 from django.core.management.base import BaseCommand
 
-from ....discount.utils import fetch_active_discounts
 from ...models import Product
-from ...utils.variant_prices import update_products_discounted_price
+from ...utils.variant_prices import update_discounted_prices_for_promotion
 
 logger = logging.getLogger(__name__)
 
+DISCOUNTED_PRODUCT_BATCH = 500
```

Comment:
Did you check the memory usage here? I'm slightly worried that there may be memory issue, as `update_discounted_prices_for_promotion` looks like may fetch quite a lot of data?

Reply:
This was checked in the task `update_products_discounted_prices_for_promotion_task` that is calling the same function.